Stormpath/Networking/Logger.swift

```diff
@@ -11,69 +11,64 @@ import Foundation
 // Simple logging class flavored for this SDK
 
 enum LogLevel {
-    case None
-    case Error
-    case Debug
-    case Verbose
+    case none
+    case error
+    case debug
+    case verbose
 }
 
 final class Logger {
+	
+	static let logLevels: [Bool:LogLevel] = [false: .debug, true: .none]
+	static let logLevel = logLevels[_isDebugAssertConfiguration()]!
     
-    static var logLevel: LogLevel {
-        if _isDebugAssertConfiguration() {
```

Comment:
Xcode 8 complains about this line, since it seems that _isDebugAssertConfiguration() is now evaluated at compile-time, so it warns that this either line 24 or 26 will never be executed.